packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -72,6 +72,14 @@ class DraggableScrollableController extends ChangeNotifier {
     return _attachedController!.extent.sizeToPixels(size);
   }
 
+  /// Returns Whether any [DraggableScrollableController] objects have attached themselves to the
+  /// [DraggableScrollableSheet].
+  ///
+  /// If this is false, then members that interact with the [ScrollPosition],
+  /// such as [sizeToPixels], [size], [animateTo], and [jumpTo], must not be
+  /// called.
+  bool get isAttached => _attachedController != null && _attachedController!.hasClients;
```

Comment:
Can you update the _assertAttached function to use this getter now?

Reply:
Done